### ISSUE TITLE
 worldeditadditions //count mising textures

### ISSUE THREAD

**Issue by Bastrabun** [sbrl/Minetest-WorldEditAdditions#90]
Repro:

1. Create a Worldedit area either with //1 and //2 or with the wand
2. Do //count (this invokes worldeditadditons)

No crash, but:

```
2023-01-21 15:01:03: ERROR[Main]: generateImage(): Could not load image "worldeditadditions_bg.png" while building texture; Creating a dummy image
2023-01-21 15:01:03: ERROR[Main]: generateImage(): Could not load image "worldeditadditions_l0.png" while building texture; Creating a dummy image
2023-01-21 15:01:03: ERROR[Main]: generateImage(): Could not load image "worldeditadditions_r2.png" while building texture; Creating a dummy image
2023-01-21 15:01:03: ERROR[Main]: generateImage(): Could not load image "worldeditadditions_bg.png" while building texture; Creating a dummy image
2023-01-21 15:01:03: ERROR[Main]: generateImage(): Could not load image "worldeditadditions_l0.png" while building texture; Creating a dummy image
2023-01-21 15:01:03: ERROR[Main]: generateImage(): Could not load image "worldeditadditions_r1.png" while building texture; Creating a dummy image
```

For my reference #3583

**Comment by sbrl** [comment]
Many thanks for this, @Bastrabun!

How did you install WorldEditAdditions please?

I recommend you use the latest release, as the latest master - while usable (I haven't seen the above bug before) - is currently a work in progress.

**Comment by Bastrabun** [comment]
Yes, it was then current main and the issue persists in current main https://github.com/sbrl/Minetest-WorldEditAdditions/commit/8652ad6335f2d1576af0f99f759f073496358ad5

**Comment by sbrl** [comment]
Ok, so I apologise for letting this bug sit - I started a new position shortly before this bug was posted and it has taken me until now to get a handle on my new schedule. I have investigated this issue on the following environment:

- **OS:** Ubuntu 23.04
- **Minetest:** 5.7.0
- **uname -a:** `Linux PC_NAME 6.2.0-20-generic #20-Ubuntu SMP PREEMPT_DYNAMIC Thu Apr  6 07:48:48 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux`
- **World type:** local

....but I'm unable to reproduce the error. Could you give me your OS on both client and server, Minetest server version (both client and server), and `uname -a` on both client and server if appropriate please?
